yarn.lock

```diff
@@ -2935,9 +2935,9 @@ camelcase@^5.0.0, camelcase@^5.2.0, camelcase@^5.3.1:
   integrity sha512-L28STB170nwWS63UjtlEOE3dldQApaJXZkOI1uMFfzf3rRuPegHaHesyee+YxQ+W6SvRDQV6UrdOdRiR153wJg==
 
 caniuse-lite@^1.0.30001165:
-  version "1.0.30001165"
-  resolved "https://registry.yarnpkg.com/caniuse-lite/-/caniuse-lite-1.0.30001165.tgz#32955490d2f60290bb186bb754f2981917fa744f"
-  integrity sha512-8cEsSMwXfx7lWSUMA2s08z9dIgsnR5NAqjXP23stdsU3AUWkCr/rr4s4OFtHXn5XXr6+7kam3QFVoYyXNPdJPA==
+  version "1.0.30001239"
+  resolved "https://registry.npmjs.org/caniuse-lite/-/caniuse-lite-1.0.30001239.tgz"
```

Comment:
This is now the only package in the lockfile that resolves from `registry.npmjs.org` instead of `registry.yarnpkg.com` - not sure how that happened or whether it is problematic :thinking:

Reply:
Hmm, I followed the official instructions of how to update caniuse. That instruction was probably not targeted for yarn users. However, I'd hope that it shouldn't be problematic :shrug:

Reply:
However, I can try to update it again with yarn. Shouldn't be a big deal, either. Will report back in a minute or so ^^

Reply:
I had to "fix" it manually. But yarn seems to be happy about it. If the CI succeeds, it should be alright, I guess.